src/main/java/com/google/sps/data/Election.java

```diff
@@ -229,9 +176,7 @@ public EmbeddedEntity createPollingStation(
    * @return true if contests and Referendums contain elements, false otherwise
```

Comment:
update documentation a bit. Should mention divisions now too.

Reply:
done